### ISSUE TITLE
Unable to use the control in Winforms net 5 (1.0.781-prerelease)

### ISSUE THREAD

**Issue by LiorBanai** [MicrosoftEdge/WebView2Feedback#896]
**Description**
Unable to use the control in Winforms net 5. Getting the expcetion below

**Version**
SDK: 
```
	<PackageReference Include="Microsoft.Web.WebView2" Version="1.0.781-prerelease" />
```
Runtime: ?
Framework: WinForms
OS: Win 10 Enterprise 2016 LTSB (1607) - 14393.4169

**Screenshots**
![image](https://user-images.githubusercontent.com/36262995/106858216-fdf3b900-66c9-11eb-8463-8f3c94faa488.png)

[AB#27485378](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/27485378)

**Comment by Zheng-Li01** [comment]
Same as the issue: https://github.com/MicrosoftEdge/WebView2Feedback/issues/895

**Comment by LiorBanai** [comment]
@Zheng-Li01 , Not sure. This is different Nuget package.

**Comment by champnic** [comment]
This looks like the same root cause as #895, but I'll keep it open as separate for now. We should have a fix available in our next SDK prerelease package. In the meantime, this error should be design-time only and the control should continue to work in your app. Thanks!

**Comment by LiorBanai** [comment]
This is newly created project so that you he control is not yet placed in the form. As soon as I drag it to the form the error pops up unfortunately.

**Comment by champnic** [comment]
This should be fixed in the latest 1.0.790-prerelease package. Thanks!
